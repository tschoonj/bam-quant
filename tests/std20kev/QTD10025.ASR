File: C:\xrfdaten\2014KW30\chris\std20kev\std10025.spe
Start: 21.07.2014 18:05:24
Stop: 21.07.2014 18:07:05
BESSY: 298.2602189 298.0993703
IONI: 3.879634E-11 3.765948E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 87875
Run: 100000
Dead: 12
ICR: 36370
OCR: 0
Accumulated BESSY: 59648.6304212
Accumulated Ioni: 7.6459294E-09
N Accumulation: 200
Normfactor:  2.9766952e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mn
exp_x 131.260406494141 131.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  25   1      5.895   2677596.      1636.     119.23
