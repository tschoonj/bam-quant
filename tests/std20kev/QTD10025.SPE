$DATA:
0 2000
20281
13126
5497
1633
304
46
11
1
3
2
1
1
0
1
1
1
115
801
922
839
837
779
740
684
718
710
695
681
608
614
635
571
569
556
594
572
541
549
494
519
509
537
484
489
471
484
432
427
453
415
468
411
485
433
453
415
399
453
447
433
439
403
416
359
417
411
381
407
370
372
395
394
380
396
409
379
372
383
357
360
351
363
369
405
353
353
378
370
376
363
391
331
400
399
390
426
369
382
366
394
448
458
388
403
396
379
392
402
353
404
398
354
345
344
377
391
357
345
371
344
396
353
349
339
333
389
354
363
366
351
395
390
327
370
320
327
345
355
331
339
355
391
370
411
440
576
589
535
482
446
440
388
364
348
386
357
376
369
414
383
409
356
369
387
369
367
354
381
375
375
365
376
342
383
369
392
377
382
374
392
394
405
426
434
387
419
429
375
405
405
406
383
448
401
369
444
422
458
514
560
680
955
1297
1525
1839
2085
1917
1717
1369
991
759
596
510
491
489
463
473
457
445
458
449
501
519
514
482
505
496
497
518
521
590
611
683
744
761
723
681
686
665
636
599
606
649
648
661
680
657
690
718
742
743
734
756
785
691
738
742
730
748
788
762
799
800
789
773
849
938
882
980
997
1025
1165
1203
1160
1330
1463
1521
1702
1875
2095
2611
3266
4631
7858
15180
29692
56358
99682
158910
229601
296974
343684
355938
328121
269859
198983
130324
77234
40807
19755
9017
4138
2057
1494
1256
1187
1119
1198
1265
1384
1625
1946
2651
3954
6241
10062
16235
24892
34516
44430
52213
55003
52640
45515
35302
25121
15964
9128
4722
2307
963
381
148
63
45
35
36
33
25
28
23
29
12
25
24
21
19
22
22
25
21
16
14
14
26
13
12
11
11
18
13
16
18
12
12
6
13
17
20
21
13
13
10
11
9
14
11
8
16
10
11
11
11
11
15
13
7
13
1
14
7
14
8
13
12
9
16
10
17
12
15
11
12
9
11
17
9
8
11
12
13
12
11
6
6
10
9
6
4
10
18
19
30
29
40
59
58
89
72
65
56
40
34
23
16
18
13
11
17
11
9
3
8
7
13
9
6
5
7
8
7
11
11
12
5
8
8
12
14
10
9
10
8
6
6
10
8
10
11
17
14
18
20
20
12
19
11
13
14
15
3
12
4
12
9
7
10
9
6
13
8
13
11
12
12
10
12
14
13
15
20
12
7
11
10
5
10
6
3
12
8
5
7
12
11
7
13
11
20
12
11
19
10
9
12
20
12
13
5
10
10
10
12
11
11
8
11
13
16
10
22
11
16
14
34
30
59
84
118
155
230
344
393
448
532
465
458
425
348
281
211
140
109
75
40
28
25
15
23
19
29
32
33
38
72
114
175
243
335
478
599
714
753
782
754
707
597
492
354
267
190
123
73
53
37
24
17
11
13
12
13
25
27
29
47
59
94
125
168
177
221
263
275
276
234
257
240
201
180
158
120
102
68
59
57
31
29
19
21
10
13
10
7
8
7
11
14
13
17
17
22
25
26
28
20
20
11
9
12
3
8
4
4
4
4
3
6
2
6
2
4
5
0
3
3
0
0
0
1
3
0
4
1
2
1
1
0
0
1
2
0
1
3
2
0
0
2
0
1
1
3
1
2
2
3
0
5
4
2
4
2
3
2
4
3
1
2
2
4
1
0
1
0
1
1
0
1
0
1
2
0
0
0
4
0
0
4
0
2
2
0
2
0
3
4
1
4
7
3
2
4
4
3
2
2
2
1
3
3
2
1
2
0
0
2
0
1
2
2
0
4
1
0
1
5
3
1
2
5
9
2
9
10
25
20
26
29
29
44
42
44
41
40
24
25
28
20
11
8
7
6
2
2
4
1
2
2
1
4
0
2
0
3
1
0
2
1
0
1
1
3
0
2
0
2
3
2
1
1
0
1
4
1
0
2
1
1
4
1
2
3
2
2
2
3
1
1
4
5
2
3
3
4
3
3
2
6
4
3
4
6
4
1
6
2
2
2
2
2
6
5
5
3
3
6
5
5
2
7
10
11
11
11
13
10
15
7
10
8
7
8
8
7
3
6
7
4
4
8
1
2
5
8
6
6
4
4
10
6
9
6
9
12
9
7
8
11
12
11
8
11
9
17
15
14
14
14
8
8
11
18
12
10
17
14
14
22
14
12
16
20
23
18
24
24
20
28
28
16
22
30
20
31
29
33
34
30
43
35
34
41
40
43
47
46
48
39
49
52
54
62
37
49
47
51
46
70
53
64
57
54
60
71
73
91
76
81
74
101
85
105
121
115
132
148
147
139
159
180
146
152
149
146
144
144
128
123
111
98
91
95
67
54
51
36
32
26
16
17
9
7
7
6
6
1
2
1
1
2
1
1
1
0
0
1
0
0
0
0
0
1
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
2
0
0
1
0
0
1
0
0
0
0
1
0
0
1
0
0
0
1
1
0
1
0
0
1
0
2
1
1
0
1
0
0
1
0
1
0
1
4
3
0
1
0
1
2
2
1
0
1
0
0
0
1
1
0
2
0
0
1
0
0
1
0
1
1
0
2
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10025.spe
Start: 21.07.2014 18:05:24
Stop: 21.07.2014 18:07:05
BESSY: 298.2602189 298.0993703
IONI: 3.879634E-11 3.765948E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 87875
Run: 100000
Dead: 12
ICR: 36370
OCR: 0
Accumulated BESSY: 59648.6304212
Accumulated Ioni: 7.6459294E-09
N Accumulation: 200
Normfactor:  2.9766952e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mn
 
exp_x 131.260406494141 131.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  25   1      5.895   2677596.      1636.     119.23
