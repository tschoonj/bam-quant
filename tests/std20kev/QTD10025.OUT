     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:39  บ   
     บ  Spectrum: QTD10025.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =    73.8   last change =   .02%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -52.1 ๑   100.0   บ     -52.1 ๑      .7   บ   
     บ  GAIN (eV/ch)  บ     20.316 ๑   2.000   บ    20.316 ๑    .002   บ   
     บ det NOISE (eV) บ      -69.6 ๑    40.0   บ     -69.6 ๑    -4.6   บ   
     บ  FANO factor   บ       .208 ๑    .050   บ      .208 ๑    .004   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Mn-Ka                           2677596. ๑   1636.           บ   
     บ    KA1         5.895   1.00000     2677596. ๑   1636.    119.2  บ   
     บ              292.736    146.00         133.           8.96E-01  บ   
     บ                                                                 บ   
     บ  2 Mn-Kb                            441071. ๑    665.           บ   
     บ    KB1         6.491   1.00000      441071. ๑    665.     48.3  บ   
     บ              322.073    154.79         122.           9.21E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           10267. ๑    105.           บ   
     บ    L1N3       11.790    .73692        7566. ๑     78.     42.0  บ   
     บ              582.908    217.90          76.           9.86E-01  บ   
     บ    L1N2       12.386    .24304        2495. ๑     27.     23.2  บ   
     บ              612.245    223.89          68.           9.84E-01  บ   
     บ    L1O3       12.982    .02004         206. ๑      8.      1.3  บ   
     บ              641.582    229.72          61.           9.86E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
